src/plot/series/whisker-series.js

```diff
@@ -0,0 +1,195 @@
+  const hasYWhiskers = positiveYVariance && cy - r > positiveYVariance;
+  if (!hasXWhiskers && !hasYWhiskers) {
+    return null;
+  }
+
+  const styleAttr = {
+    opacity: opacityFunctor ? opacityFunctor(d) : DEFAULT_OPACITY,
+    stroke: strokeFunctor && strokeFunctor(d),
+    strokeWidth: strokeWidth || DEFAULT_STROKE_WIDTH,
+    ...style
+  };
+  const crossBarExtension = crossBarWidth / 2;
+
+  const rightLineAttrs = {
+    x1: cx + r,
+    y1: cy,
+    x2: positiveXVariance,
+    y2: cy,
+    style: styleAttr
+  };
+  const leftLineAttrs = {
+    x1: cx - r,
+    y1: cy,
+    x2: negativeXVariance,
+    y2: cy,
+    style: styleAttr
+  };
+  const rightCrossBarAttrs = {
+    x1: positiveXVariance,
+    y1: cy - crossBarExtension,
+    x2: positiveXVariance,
+    y2: cy + crossBarExtension,
+    style: styleAttr
+  };
+  const leftCrossBarAttrs = {
+    x1: negativeXVariance,
+    y1: cy - crossBarExtension,
+    x2: negativeXVariance,
+    y2: cy + crossBarExtension,
+    style: styleAttr
+  };
+
+  const upperLineAttrs = {
+    x1: cx,
+    y1: cy - r,
+    x2: cx,
+    y2: positiveYVariance,
+    style: styleAttr
+  };
+  const lowerLineAttrs = {
+    x1: cx,
+    y1: cy + r,
+    x2: cx,
+    y2: negativeYVariance,
+    style: styleAttr
+  };
+  const upperCrossBarAttrs = {
+    x1: cx - crossBarExtension,
+    y1: positiveYVariance,
+    x2: cx + crossBarExtension,
+    y2: positiveYVariance,
+    style: styleAttr
+  };
+  const lowerCrossBarAttrs = {
+    x1: cx - crossBarExtension,
+    y1: negativeYVariance,
+    x2: cx + crossBarExtension,
+    y2: negativeYVariance,
+    style: styleAttr
+  };
+
+  return (
+    <g className="mark-whiskers" key={i}
+      onClick={e => valueClickHandler(d, e)}
+      onMouseOver={e => valueMouseOverHandler(d, e)}
+      onMouseOut={e => valueMouseOutHandler(d, e)}
+    >
+      {hasXWhiskers ?
+        <g className="x-whiskers">
+          <line {...rightLineAttrs} />
+          <line {...leftLineAttrs} />
+          <line {...rightCrossBarAttrs} />
+          <line {...leftCrossBarAttrs} />
+        </g> :
+        null
+      }
+      {hasYWhiskers ?
+        <g className="y-whiskers">
+          <line {...upperLineAttrs} />
+          <line {...lowerLineAttrs} />
+          <line {...upperCrossBarAttrs} />
+          <line {...lowerCrossBarAttrs} />
+        </g> :
+        null
+      }
+    </g>
+  );
+};
+
+class WhiskerSeries extends AbstractSeries {
+  render() {
+    const {
+      animation, className, crossBarWidth, data, marginLeft, marginTop, strokeWidth,
+      style
+    } = this.props;
+    if (!data) {
+      return null;
+    }
+    if (animation) {
+      return (
+        <Animation {...this.props} animatedProps={ANIMATED_SERIES_PROPS}>
+          <WhiskerSeries {...this.props} animation={null}/>
+        </Animation>
+      );
+    }
+
+    const whiskerMarkProps = {
+      crossBarWidth: crossBarWidth || DEFAULT_CROSS_BAR_WIDTH,
```

Comment:
if you have the defaultProps then you dont need to this default here, (as it will always be appropriate supplied), so make this item say

```
crossBarWidth,
```